kuma/static/js/contribution-handler.js

```diff
@@ -0,0 +1,513 @@
+
+        } else {
+            // Reset radio when selecting custom amount.
+            form.find('input[type=\'radio\']:checked').prop('checked', false);
+        }
+
+        selectedAmount = (Math.floor(event.target.value * 100) / 100);
+
+        var newValue = (selectedAmount < 1 || isNaN(selectedAmount)) ? '' : '$' + selectedAmount;
+
+        amount.html(newValue);
+    }
+
+    /**
+     * Set the error message for any required field.
+     * @param {Element} field Field element.
+     */
+    function setFieldError(field) {
+        $(field).addClass('error');
+        $(field).next('.errorlist').remove();
+
+        var error = $(field).attr('data-error-message');
+
+        $('<ul class="errorlist"><li>' + error + '</li></ul>').insertAfter($(field));
+
+        if ($(field).is('#id_donation_amount')) {
+            mdn.analytics.trackEvent({
+                category: 'Contribution popover',
+                action: 'Invalid amount selected',
+                value: 1
+            });
+        }
+    }
+
+    /**
+     * Clear the error message for any required field.
+     * @param {Element} field Field element.
+     */
+    function clearFieldError(field) {
+        $(field).removeClass('error');
+        $(field).next('.errorlist').remove();
+    }
+
+    /**
+     * Checks field validity and sets any errors if required.
+     * @param {jQuery.Event} event Event object.
+     */
+    function onChange(event) {
+        var field = $(event.target)[0];
+
+        if (field.checkValidity()) {
+            clearFieldError(field);
+        } else {
+            setFieldError(field);
+        }
+    }
+
+    /**
+     * Handles the form submit. Checks for field validity using built in browser
+     * checkValidity functionality and then attempts to open the Stripe modal if
+     * the form is valid.
+     */
+    function onSubmit() {
+        // FE form validation.
+        var valid = form[0].checkValidity();
+
+        if (!valid || (selectedAmount < 1 || isNaN(selectedAmount))) {
+            if (emailField[0].checkValidity()) {
+                clearFieldError(emailField[0]);
+            } else {
+                setFieldError(emailField[0]);
+            }
+
+            if (nameField[0].checkValidity()) {
+                clearFieldError(nameField[0]);
+            } else {
+                setFieldError(nameField[0]);
+            }
+
+            if (selectedAmount >= 1) {
+                clearFieldError(customAmountInput);
+            } else {
+                setFieldError(customAmountInput);
+            }
+
+            return;
+        }
+
+        // Send GA Event.
+        mdn.analytics.trackEvent({
+            category: 'Contribution submission',
+            action: isPopoverBanner ? 'On Popover' : 'On Page',
+            value: 1
+        });
+
+        // On success open Stripe Checkout modal.
+        stripeHandler.open({
+            image: 'https://avatars1.githubusercontent.com/u/7565578?s=280&v=4',
```

Comment:
This should be the white-on-black logo, and sourced from MDN. I think this means it becomes another ``mdn.`` item in the header JS.

Reply:
I realized this was the image for https://github.com/mdn/, and I changed it there.